src/Symfony/Component/Validator/Tests/ConstraintValidatorTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+final class ConstraintValidatorTest extends TestCase
+{
+    /**
+     * @dataProvider formatValueProvider
+     */
+    public function testFormatValue($expected, $value, $format = 0)
+    {
+        $this->assertSame($expected, (new TestFormatValueConstraintValidator())->formatValueProxy($value, $format));
+    }
+
+    public function formatValueProvider(): array
```

Comment:
almost done ;)

Reply:
Damn I should install PHP 5 locally 😅

Reply:
It can be configurable via PHPStorm: "PHP language level"